NEWS.md

```diff
@@ -1,3 +1,7 @@
+# admiral 1.0.2
+
+- Fix bug in `derive_param_tte()` where argument `dataset` populated and `PARAMCD` in `set_values_to` argument is an expression. (#2336)
```

Comment:
Hey Gordon - I guess I am still confused about what we are telling the user here, what we are fixing and there might be a dose of me not fully following  :)

The bug is from the function picking up potential false positives around the parameters and falling over with the duplication error?

The fix is now we check to make sure the parameters do not already exist in the datasets and give feedback to the user?

I feel like reading the issue and news bullet has only confused me!!

Reply:
Previously the check was looking at the contents of what `PARAMCD` was in `set_values_to` argument, and seeing if the `PARAMCD` already existed in dataset that is passed into `dataset` argument.  If `PARAMCD` was not an expression ie `PARAMCD = "XYZ"` then check worked. However, check was very early in function, so if `PARAMCD` to be created was an expression, and wasn't resolved yet, this caused the ERROR. I moved the check to near the end, where `PARAMCD` is resolved in the dataset holding the new params. Happy to change the wording in the NEWS item!

Reply:
ooohhh could that be the new summary just a little more polished please.

thanks again for digging into this and ELI5 to me!!